components/sensor/vl53l1x.rst

```diff
@@ -0,0 +1,70 @@
+VL53L1X Time Of Flight Distance Sensor
+======================================
+
+.. seo::
+    :description: Instructions for setting up VL53L1X distance sensors in ESPHome.
+    :image: vl53l1x.jpg
+    :keywords: VL53L1X
+
+The ``vl53l1x`` sensor platform allows you to use VL53L1X optical time of flight
+(`datasheet <https://www.st.com/resource/en/datasheet/vl53l1x.pdf>`__,
+`Pololu <https://www.pololu.com/product/3415/resources>`__) with ESPHome
+to measure distances. The sensor works optically by emitting short infrared pulses
+and measuring the time it takes the light to be reflected back.
+
+The sensor can measure distances up to 4 meters, though that figure depends significantly
+on several conditions like surface reflectance, field of view, temperature etc.
+
+The :ref:`I²C Bus <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/vl53l1x.jpg
+    :align: center
+    :width: 30.0%
+
+    VL53L1X Time Of Flight Distance Sensor.
+
+.. figure:: images/vl53l1x-ui.png
+    :align: center
+    :width: 80.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: vl53l1x
+        name: "VL53L1x Distance"
+        update_interval: 60s
+        distance_mode: LONG
+        timing_budget: 50ms
```

Comment:
```suggestion
.. code-block:: yaml

    sensor:
      - platform: vl53l1x
        name: "VL53L1x *name*"
        update_interval: 60s
        distance_mode: LONG
        timing_budget: 50ms

```

Reply:
I can't commit your suggestion because github complains about "Suggestion cannot be identical to original text."

What is the actual change?